### PR TITLE
chore: update packages/example deployment script and cleanup package/example

### PR DESCRIPTION
**Pull request by emilyrohrbough**
Fixing deployment issues with `packages/example`, cleaning up dead code, auto-fix lint changes and updated README to reflect current usage.

### CODE REVIEW

File: npm/create-cypress-tests/scripts/examples/cypress/support/e2e.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/e2e.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
```
Comment (lmiller1990) [inline npm/create-cypress-tests/scripts/examples/cypress/support/e2e.js:5]: I wonder if we should deprecate this library, it has a total for 44 weekly downloads: https://www.npmjs.com/package/create-cypress-tests?activeTab=versions

File: packages/example/bin/build.js [hunk 2]
```diff
@@ -1,28 +1,29 @@
 #!/usr/bin/env node
 
+/* eslint-disable no-console */
```
Comment (lmiller1990) [inline packages/example/bin/build.js:3]: Did we need this? I don't see any `console.log` added here.

**Review by lmiller1990 — APPROVED** [review]
Looks good! :fire:

File: packages/example/bin/build.js (hunk 2 above)
Comment (emilyrohrbough) [reply packages/example/bin/build.js:3]: see lines 31 & 34

File: packages/example/README.md [hunk 3]
```diff
@@ -1,27 +1,30 @@
+# Example
 
-## Scaffold config files 
+This package is responsible for coping the `cypress/e2e` and `app` files from [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink) into the cypress repository.
 
-The `cypress/plugins/index.js`, `cypress/support/*` and `cypress/tsconfig.json` from this package are used for user scaffolding in `packages/server` and `npm/create-cypress-tests`. This configuration files are by default injected when user instals Cypress.
+The `cypress/e2e` tests, pulled into this package from the [kitchen sink app](https://github.com/cypress-io/cypress-example-kitchensink, are used for scaffolding user's e2e tests in `packages/data-context` and in `npm/create-cypress-tests`.
 
-## Examples
+The `app` content, pulled into this package from the [kitchen sink app](https://github.com/cypress-io/cypress-example-kitchensink, are published to `cypress-io/cypress` repository's Github page, [https://example.cypress.io](https://example.cypress.io).
```
Comment (AtofStryker) [inline packages/example/README.md]:
```suggestion
The `app` content, pulled into this package from the [kitchen sink app](https://github.com/cypress-io/cypress-example-kitchensink), is published to `cypress-io/cypress` repository's Github page, [https://example.cypress.io](https://example.cypress.io).
```

File: packages/example/README.md [hunk 4]
```diff
@@ -1,27 +1,30 @@
+# Example
 
-## Scaffold config files 
+This package is responsible for coping the `cypress/e2e` and `app` files from [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink) into the cypress repository.
 
-The `cypress/plugins/index.js`, `cypress/support/*` and `cypress/tsconfig.json` from this package are used for user scaffolding in `packages/server` and `npm/create-cypress-tests`. This configuration files are by default injected when user instals Cypress.
+The `cypress/e2e` tests, pulled into this package from the [kitchen sink app](https://github.com/cypress-io/cypress-example-kitchensink, are used for scaffolding user's e2e tests in `packages/data-context` and in `npm/create-cypress-tests`.
```
Comment (AtofStryker) [inline packages/example/README.md]:
```suggestion
The `cypress/e2e` tests, pulled into this package from the [kitchen sink app](https://github.com/cypress-io/cypress-example-kitchensink), are used for scaffolding user's e2e tests in `packages/data-context` and in `npm/create-cypress-tests`.
```

File: packages/example/README.md [hunk 5]
```diff
@@ -1,27 +1,30 @@
+# Example
 
-## Scaffold config files 
+This package is responsible for coping the `cypress/e2e` and `app` files from [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink) into the cypress repository.
```
Comment (AtofStryker) [inline packages/example/README.md]:
```suggestion
This package is responsible for copying the `cypress/e2e` and `app` files from [`cypress-example-kitchensink`](https://github.com/cypress-io/cypress-example-kitchensink) into the cypress repository.
```

File: packages/example/README.md [hunk 6]
```diff
@@ -1,27 +1,30 @@
...
 
-## Updating the `example` app
+## Major Version Bumps of Cypress
+
+If any of the breaking changes in the next major release requires updates to Cypress commands or APIs, verify the site content in `cypress-example-kitchensink` is up-to-date and that all examples that will be scaffolded can successfully run with the breaking change.
 
-After [releasing a new version](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/CONTRIBUTING.md#deployment) on the [`cypress-example-kitchen-sink` repo](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/cypress/e2e/2-advanced-examples), you now want to update the `example`'s dependency to match the newly released version.
+## Using a new version of `cypress-example-kitchen-sink`
+
+After [new version is automatically](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/CONTRIBUTING.md#deployment) on the [`cypress-example-kitchen-sink` repo](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/cypress/e2e/2-advanced-examples) when a commit is merged to `master`, you will need to update the `example`'s dependency to match the newly released version.
```
Comment (AtofStryker) [inline packages/example/README.md]: This reads kind of strange to me and I'm struggling to grok it. How is something like this?

```suggestion
When a commit is merged into `master`, a new version of the [`cypress-example-kitchen-sink` repo](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/cypress/e2e/2-advanced-examples) is released [automatically to npm](https://github.com/cypress-io/cypress-example-kitchensink/blob/master/CONTRIBUTING.md#deployment). When this happens, you will need to update the `example`'s dependency to match the newly released version.
```

**Review by AtofStryker — CHANGES_REQUESTED** [review]

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**